### ISSUE TITLE
Crashing when Discord Webhook messages are sent to the channel

### ISSUE THREAD

**Issue by Psykus0** [reactiflux/discord-irc#146]
Hi,

I'm using the webhook feature of Discord to send messages from IFTTT to a Discord channel, and whenever they are sent, discord-irc crashes and the message does not display in the IRC channel before discord-irc disconnects.

https://support.discordapp.com/hc/en-us/articles/228383668-Intro-to-Webhooks

I got the exact error message below:
```js
/usr/lib/node_modules/discord-irc/dist/bot.js:156
      const nickname = userDetails.nickname || author.username;
                                  ^

TypeError: Cannot read property 'nickname' of null
    at Bot.sendToIRC (/usr/lib/node_modules/discord-irc/dist/bot.js:156:35)
    at Client.discord.on.message (/usr/lib/node_modules/discord-irc/dist/bot.js:108:12)
    at emitOne (events.js:96:13)
    at Client.emit (events.js:188:7)
    at MessageCreateHandler.handle (/usr/lib/node_modules/discord-irc/node_modules/discord.js/src/client/websocket/packets/handlers/MessageCreate.js:9:34)
    at WebSocketPacketManager.handle (/usr/lib/node_modules/discord-irc/node_modules/discord.js/src/client/websocket/packets/WebSocketPacketManager.js:93:65)
    at WebSocketManager.eventMessage (/usr/lib/node_modules/discord-irc/node_modules/discord.js/src/client/websocket/WebSocketManager.js:198:31)
    at WebSocket.ws.onmessage (/usr/lib/node_modules/discord-irc/node_modules/discord.js/src/client/websocket/WebSocketManager.js:74:37)
    at WebSocket.onMessage (/usr/lib/node_modules/discord-irc/node_modules/ws/lib/WebSocket.js:442:14)
    at emitTwo (events.js:106:13)
```

I installed discord-irc through npm and am on version 3.10.8 according to the output of "npm -v discord-irc"

Let me know if you need any info or help reproducing the issue and I'd be glad to help.

**Comment by ekmartin** [comment]
I guess there's two options here, either filter them out or send them to IRC with the webhook's name as username (i.e. `<GitHub> New commit to...`). 

I think I like the last option best, but not sure. What do you think?

**Comment by Psykus0** [comment]
Yeah the last option sounds great.

**Comment by Psykus0** [comment]
This seems fixed in the newest version on GitHub :+1:

**Comment by ekmartin** [comment]
Cool, then it got fixed in #147.
